tests/phpunit/tests/functions/wpOpcacheInvalidateDirectory.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Tests wp_opcache_invalidate_directory().
+ *
+ * @group functions.php
```

Comment:
I believe `tests/functions/` contains tests for functions located in `wp-includes/functions.php`.

Since these two functions are located in `wp-admin/includes/file.php` and are a part of the Filesystem API, perhaps `tests/fiesystem/` would be a better location for these tests?

Reply:
Right you are! Not sure how I managed that lol. Will update shortly.

Reply:
Fixed in [11ae273](https://github.com/WordPress/wordpress-develop/pull/3909/commits/11ae273cd1fe9571031045d750675454575ed85a). Thanks!